Ports/thesilversearcher/package.sh

```diff
@@ -2,7 +2,7 @@
 port=thesilversearcher
 version=2.2.0
 useconfigure="true"
-files="https://geoff.greer.fm/ag/releases/the_silver_searcher-${version}.tar.gz the_silver_searcher-${version}.tar.xz d9621a878542f3733b5c6e71c849b9d1a830ed77cb1a1f6c2ea441d4b0643170"
+files="https://geoff.greer.fm/ag/releases/the_silver_searcher-${version}.tar.gz the_silver_searcher-${version}.tar.gz d9621a878542f3733b5c6e71c849b9d1a830ed77cb1a1f6c2ea441d4b0643170"
```

Comment:
Please also include this fix for [awk](https://github.com/SerenityOS/serenity/blob/3a611ff2cbf351fc12399e010d188ad0d7e9f7dd/Ports/awk/package.sh#L5). It was annoying me but I didn't want to open a PR for 2 single letter changes.